### PR TITLE
Add template for previewing URL

### PR DESCRIPTION
**Pull request by eddietejeda**
## Changes proposed in this pull request:
- Add template for federalist preview urls
 
## security considerations
None

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -3,5 +3,8 @@
 -
 -
 
-## security considerations
+:sunglasses:[PREVIEW URL](https://cg-88d42ca6-59d7-47e0-9500-4dd9251360b9.app.cloud.gov/preview/cloud-gov/cg-site/BRANCH_NAME)
```
Comment (bengerman13) [inline .github/pull_request_template.md:6]: is the intent here that a user should be manually replacing `BRANCH_NAME` with the name of their branch?

Comment (eddietejeda) [reply .github/pull_request_template.md:6]: Correct. 

Also how its done in other parts of TTS: https://github.com/18F/18f.gsa.gov/pull/3151

If you can figure out a way to inject the branch name as a variable into the PULL_REQUEST template, you'll make a lot of friends here.

Comment (bengerman13) [reply .github/pull_request_template.md:6]: gotcha.

**Comment by bengerman13** [comment]
is this really required with the federalist build details? I'm happy to add it in if it's useful, but I wonder if it's easier to just nudge folks towards the sweet integrations Federalist already built

**Review by bengerman13 — APPROVED** [review]

**Comment by eddietejeda** [comment]
With preview url you can link directly to the page you want to test. I think its worth it.

**Pull request merged**